.github/ISSUE_TEMPLATE/--question.md

```diff
@@ -0,0 +1,9 @@
+---
+name: "❓ Question"
+about: Please use https://discuss.lightbend.com/c/akka/ for questions
+
+---
+
+If you have a [Lightbend Subscription](https://www.lightbend.com/lightbend-platform-subscription), please reach out via the [Lightbend Portal](https://portal.lightbend.com/).
+
+Otherwise, please use the [Lightbend discuss forum](https://discuss.lightbend.com/c/akka/) for questions instead of posting them to the issue tracker.
```

Comment:
```suggestion
Otherwise, please use the [Lightbend discuss forum](https://discuss.akka.io/) for questions instead of posting them to the issue tracker.
```